ui/src/containers/ConfirmCollectDialog.jsx

```diff
@@ -303,14 +303,14 @@ export class ConfirmCollectDialog extends React.Component {
             </thead>
             <tbody id="table-body">
               {tasks.map((task) => {
-                let { parameters } = task;
+                const { parameters } =
+                  task.type === 'Interleaved'
+                    ? task.parameters.wedges[0]
+                    : task;
+
                 const sample = this.props.sampleGrid.sampleList[task.sampleID];
                 const sampleName = `${sample.sampleName} - ${sample.proteinAcronym}`;
 
-                if (task.type === 'Interleaved') {
-                  parameters = task.parameters.wedges[0].parameters;
```

Comment:
I've relaxed assignments to existing variables, so this actually passes now, but I thought the ternary was nicer in this case.